src/main/java/au/org/ala/biocache/web/DownloadController.java

```diff
@@ -118,13 +122,14 @@ public boolean apply(Object source, String name, Object value) {
             DownloadRequestParams requestParams,
             @RequestParam(value = "ip", required = false) String ip,
             @RequestParam(value = "apiKey", required = false) String apiKey,
+            @RequestParam(value = "email", required = true) String email,
```

Comment:
I am not familiar enough with the way this `@RequestParam` relates to the `DownloadRequestsParams.getEmail` value. What is the semantic difference of having it here?

Reply:
The `@RequestParam` maps the specific GET/POST request parameter to the method argument,
`@RespectBody` marshals the GET/POST request parameters to the `DownloadRequestParams` object. The value of the `DownloadContoller.occurrenceDownload` method's `email` argument and requestParams.getEmail() are the same. 

I added the `email` as a method argument to elevate its importance to the download process and flag it as a required parameter.

Reply:
Thanks, the required parameter flag is a useful addition.